### PR TITLE
kebab: Update from OOS 13.1.0.513(EX01)

### PR DESCRIPTION
**Pull request by Nixsuki**

### CODE REVIEW

File: ih8sn.conf.kebab [hunk 1]
```diff
@@ -0,0 +1,2 @@
+BUILD_FINGERPRINT=OnePlus/OnePlus8T/OnePlus8T:13/RKQ1.211119.001/R.ed8c2c_1_2:user/release-keys
```
Comment (althafvly) [inline ih8sn.conf.kebab:1]: remove this file

Comment (Nixsuki) [reply ih8sn.conf.kebab:1]: Why remove, it should be updated?

Comment (Nixsuki) [reply ih8sn.conf.kebab:1]: Updated.

https://github.com/althafvly/ih8sn/pull/6/commits/43d5062a9eaea8dc602e83ee5aa21add79654c57

Comment (althafvly) [reply ih8sn.conf.kebab:1]: you are commiting two files. remove one

**Pull request merged**